web-client/src/main/java/com/weather/web/client/dto/WeatherBitResponseDto.java

```diff
@@ -5,38 +5,5 @@
 
 import java.util.List;
 
```

Comment:
Ok, eventually to me it doesnt make a sense to remove the comment only here, plese remove comments in other files